### ISSUE TITLE
option to make swarm remember what inputs you want used when creating a new preset

### ISSUE THREAD

**Issue by fraterinchristo** [mcmonkeyprojects/SwarmUI#484]
### Feature Idea

selecting nearly 20 different inputs every time when creating a new preset is pretty annoying so having the option to make swarm remember what inputs you want used would be cool.

**Comment by fraterinchristo** [comment]
steps, cfg scale, sampler, scheduler, refiner model, refiner vae, refiner control percentage, refiner steps, refiner cfg scale, refiner method, refiner upscale, refiner upscale method, model, loras, lora weights. because i want the presets to have all the settings bc i don't want to start changing them when using different models. i know i could just make a preset with the correct settings for each model i use but i'm brain dead so i do it this way. i also don't want to forget to change them when switching between models and end up creating a monstrosity.

**Comment by fraterinchristo** [comment]
steps, cfg scale, sampler, scheduler, refiner model, refiner vae, refiner control percentage, refiner steps, refiner cfg scale, refiner method, refiner upscale, refiner upscale method, model, loras, lora weights. because i want the presets to have all the settings bc i don't want to start changing them when using different models. i know i could just make a preset with the correct settings for each model i use but i'm brain dead so i do it this way. i also don't want to forget to change them when switching between models and end up creating a monstrosity.

**Comment by mcmonkey4eva** [comment]
ftr most of the time the correct setting on half of those is _not_.
Sampler/Scheduler: usually default is recommended ie leave unchecked.
Refiner VAE, refiner Steps, Refiner CFG... same story those are for very special cases.
Refiner model is a very particular type of unusual generation.

Steps... I mean just set one global default and it's fine for 90% of models, only eg turbo needs a unique step count.
CFG Scale yeah a lil variable, but it's mostly by category (eg Flux wants CFG=1, whereas SD1/SDXL models want CFG around 7ish)

Refiner settings in general only for certain models (eg SD1 models, or particular quality-centric presets on eg SDXL). You almost certainly don't want a refiner upscale on eg modern flux gens.

-----------

In short: you're fiddling with settings a lot more than you probably should be.

**Comment by mcmonkey4eva** [comment]
https://github.com/mcmonkeyprojects/SwarmUI/issues/390 covers the relevant feature request concept already
